frontendUiTableImpl/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
   implementation project(':frontendGraphApi')
   implementation project(':frontendVfsUtils')
   implementation project(':frontendProviderServices')
+  implementation project(':frontendResourceBundles')
```

Comment:
Pls also add i18n checker, as in `frontendActions` (maybe extract that to `ext`, as done already for subtyping checker)